client/extensions/relatedArticles/src/EventsRelatedArticles/RelatedArticlesListComponent.tsx

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
```

Comment:
Same thing here, stick with the DRY principle and use `ArticleItemConcise` component from the SuperdeskAPI.
This way this one component is used in multiple places to render related articles, rather than duplication of the same functionality.

If we need changes done, such as to show language, then this should be done on the `ArticleItemConcise` component